### PR TITLE
Add setup for MMB package

### PR DESCRIPTION
**Pull request by tbergq**

### CODE REVIEW

File: app/MMB/package.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+  "name": "@kiwicom/mobile-manage-by-booking",
```
Comment (mrtnzlml) [inline app/MMB/package.json]:
```diff
-   "name": "@kiwicom/mobile-manage-by-booking",
+   "name": "@kiwicom/mobile-manage-my-booking",
```

File: app/core/package.json [hunk 2]
```diff
@@ -4,6 +4,7 @@
   "private": true,
   "dependencies": {
     "@kiwicom/react-native-app-hotels": "^0",
+    "@kiwicom/mobile-manage-by-booking": "^0",
```
Comment (mrtnzlml) [inline app/core/package.json]:
```diff
-     "@kiwicom/mobile-manage-by-booking": "^0",
+     "@kiwicom/mobile-manage-my-booking": "^0",
```

File: app/core/src/navigation/index.js [hunk 3]
```diff
@@ -30,16 +29,32 @@ const VoidStack = StackNavigator(
...
       screen: HomepageStack,
     },
     HotelsPackage: {
-      screen: withMappedProps(HotelsPackageWrapper),
+      screen: HotelsStack,
     },
-    SingleHotelPackage: {
-      screen: withMappedProps(SingleHotelsPackageWrapper),
+    MMBPackage: {
+      screen: tabNavigator,
```
Comment (mrtnzlml) [inline app/core/src/navigation/index.js]: I don't understand this change. My initial intention was to go closer to our native app by creating the tab navigator (as we have in our app) so we can say later - _"look, this is what we have and it's basically the same with native"_. So I wanted to use even the homepage as a homepage (even though we have only links there so far). What do you think about that?

File: app/MMB/src/navigation/MMBScreen.js [hunk 4]
```diff
@@ -0,0 +1,42 @@
+// @flow
+
+import * as React from 'react';
+import { View } from 'react-native';
+import { Translation } from '@kiwicom/mobile-localization';
+import type { NavigationType } from '@kiwicom/mobile-navigation';
+import { HeaderBackButton } from 'react-navigation';
+
+const isDevelopment = process.env.NODE_ENV;
```
Comment (mrtnzlml) [inline app/MMB/src/navigation/MMBScreen.js]: Just a silly question - `NODE_ENV` is defined only during development and therefore it's safe to use it like this?

File: app/MMB/src/navigation/NavigationStack.js [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+import {
+  StackNavigator,
+  StackNavigatorOptions,
+  type NavigationType,
+} from '@kiwicom/mobile-navigation';
+
+import MMBScreen from './MMBScreen';
+
+export type NavigationProps = {|
```
Comment (mrtnzlml) [inline app/MMB/src/navigation/NavigationStack.js]: This type is not used.

File: app/MMB/package.json (hunk 1 above)
Comment (tbergq) [reply app/MMB/package.json]: 🙈

File: app/core/src/navigation/index.js (hunk 3 above)
Comment (tbergq) [reply app/core/src/navigation/index.js]: Ait, I guess I can work with that.

Comment (mrtnzlml) [reply app/core/src/navigation/index.js]: The change is simple - just keep the homepages as we do (we could actually remove some links) and let's work on the second tab (the same with our native app).

Later on, we will have to introduce the horizontal scroll view with suggested hotels and we can directly use it on our homepage...

File: app/MMB/src/navigation/MMBScreen.js (hunk 4 above)
Comment (tbergq) [reply app/MMB/src/navigation/MMBScreen.js]: It should be `development` in dev-mode and `production` when deployed. 

This is just a silly developer-mistake 🙈

File: app/MMB/src/navigation/NavigationStack.js (hunk 5 above)
Comment (tbergq) [reply app/MMB/src/navigation/NavigationStack.js]: copy/paste 🙈

**Comment by tbergq** [comment]
@mrtnzlml please check again

**Pull request merged**